src/command_line_parser.cc

```diff
@@ -139,6 +139,7 @@ CLParser::Usage(const std::string& msg)
   std::cerr << "\t--request-count <number of requests>" << std::endl;
   std::cerr << "\t--warmup-request-count <number of warmup requests>"
             << std::endl;
+  std::cerr << "\t--fixed-schedule" << std::endl;
```

Comment:
Similar to [this](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2057033395), PA was accidentally missing help option documentation for fixed schedule mode.

Adding in this PR.